routes/google2fa.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+--------------------------------------------------------------------------
+Google2FA
+--------------------------------------------------------------------------
+*/
+
+// Display 2FA form if User has generated Google2FA
+Route::get('/2fa', 'PasswordSecurityController@show2faForm');
```

Comment:
As a general rule, with these routes in `settings`, I would also prefix them with `security`. So it becomes `/settings/security/2fa`, `settings/security/enable2fa` etc.